prov/tcp/src/tcpx_comm.c

```diff
@@ -76,18 +76,78 @@ int tcpx_recv_field(SOCKET sock, void *buf, size_t buf_len,
 		-FI_EAGAIN: FI_SUCCESS;
 }
 
+static int tcpx_process_read_req(struct tcpx_pe_entry *pe_entry)
+{
+	return -FI_ENODATA;
+}
+
+static int tcpx_process_read_rsp(struct tcpx_pe_entry *pe_entry)
+{
+	return -FI_ENODATA;
+}
+
+static int tcpx_process_write(struct tcpx_pe_entry *pe_entry)
+{
+	return -FI_ENODATA;
+}
+
+static int tcpx_validate_rma_data(struct tcpx_pe_entry *pe_entry)
+{
+	return -FI_ENODATA;
+}
+static int tcpx_process_rma_data(struct tcpx_pe_entry *pe_entry)
```

Comment:
I would add an 'rx' tag somewhere in these names, to indicate that this is the receive side handling of an rma.